packages/cli/src/platform/apikey/show.ts

```diff
@@ -1,4 +1,5 @@
 import { arg, Command, isError } from '@prisma/internals'
+import { table } from 'console'
```

Comment:
If you could avoid this and use `console.table`, we don't use this at all in the codebase and it's better to be consistent.